flathub.json

```diff
@@ -0,0 +1,4 @@
+{
+  "only-arches": ["x86_64"],
```

Comment:
this is disappointing.

Reply:
I know

Reply:
so remove it?

Reply:
what I am not understanding here?

from the docs:
only-arches and skip-arches are used to build the source only on the specified architecture or skip building it on the specified architecture respectively. This is often useful for binary packages.

that binary must be build on x86 only, isn't that right?

Reply:
hence you could build from source here. and it would handle aarch64. You said it was Java, so that shouldn't even be a problem of compiling just shipping the JRE.

Reply:
you said that you don't even know what an ambilight is, so what's the point?
my app does not need aarch64 currently :)

My app is not java only, it's a java app with some bindings to C libraries like gstreamer, and others.

Reply:
> you said that you don't even know what an ambilight is

you haven't explained it either.

Reply:
I'm sorry. Ambilight or Bias Light is a technology that enhances your viewing experience by projecting colored lights onto the wall behind your TV, matching the colors on the screen. This creates a more immersive and engaging atmosphere, making your favorite shows, movies, and games feel even more captivating.   

you can have an idea on what is does here:
https://youtu.be/PO37JC7-Nn4?si=0fSJXPzTQ8pZJYEF